### PR TITLE
Update version_history.md and readme.md

### PR DESCRIPTION
**Pull request by ChristinaElder**

### CODE REVIEW

File: version_history.md [hunk 1]
```diff
@@ -19,7 +21,51 @@ This file contains archived version history information for the [ROCm project](h
...
 
+### New features and enhancements in ROCm 2.7.2
 
+This release is a hotfix for ROCm release 2.7.
+
+#### Defect fixed in ROCm 2.7.2
+
+##### A defect in upgrades from older ROCm releases has been fixed.
+
+### New features and enhancements in ROCm 2.7.1
```
Comment (nelsonc-amd) [inline version_history.md]: I would drop this entire section section titled New features and enhancements in ROCm 2.7.1

File: version_history.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ## ROCm Version History
 This file contains archived version history information for the [ROCm project](https://github.com/RadeonOpenCompute/ROCm)
 
-### Current ROCm Version: 2.7.2
+### Current ROCm Version: 2.8
+- [New features and enhancements in ROCm 2.7.2](#new-features-and-enhancements-in-rocm-272)
+- [New features and enhancements in ROCm 2.7.1](#new-features-and-enhancements-in-rocm-271)
```
Comment (nelsonc-amd) [inline version_history.md]: And drop the link for the 2.7.1 section.

File: README.md [hunk 3]
```diff
@@ -715,7 +661,7 @@ If you chose a directory other than `~/bin/` to install `repo`, you should use t
 ```shell
 mkdir -p ~/ROCm/
 cd ~/ROCm/
-~/bin/repo init -u https://github.com/RadeonOpenCompute/ROCm.git -b roc-2.7.1
+~/bin/repo init -u https://github.com/RadeonOpenCompute/ROCm.git -b roc-2.8.1
```
Comment (nelsonc-amd) [inline README.md]: roc-2.8.1 should be 'roc-2.8.0'

File: README.md [hunk 4]
```diff
@@ -29,125 +27,67 @@ Please refer the ROCm Documentation [here](https://rocm-documentation.readthedoc
...
-  - [rocThrust](https://github.com/ROCmSoftwarePlatform/rocThrust/tree/master-rocm-2.7)
-  - [ROCm SMI Lib](https://github.com/RadeonOpenCompute/rocm_smi_lib/tree/roc-2.7.0)
-  - [RCCL](https://github.com/ROCmSoftwarePlatform/rccl/tree/master-rocm-2.7)
+  - [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen/tree/roc-2.8.0)
+  - [rocThrust](https://github.com/ROCmSoftwarePlatform/rocThrust/tree/master-rocm-2.8)
+  - [ROCm SMI Lib](https://github.com/RadeonOpenCompute/rocm_smi_lib/tree/roc-2.8.0)
+  - [RCCL](https://github.com/ROCmSoftwarePlatform/rccl/tree/master-rocm-2.8)
   - [MIVisionX](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/tree/1.3.0)
-  - [hipCUB](https://github.com/ROCmSoftwarePlatform/hipCUB/tree/rocm-2.7)
+  - [hipCUB](https://github.com/ROCmSoftwarePlatform/hipCUB/tree/rocm-2.8)
```
Comment (nelsonc-amd) [inline README.md]: Should be: https://github.com/ROCmSoftwarePlatform/hipCUB/tree/master-rocm-2.8 (rocm-2.8 -> master-rocm-2.8).

File: version_history.md (hunk 1 above)
Comment (ChristinaElder) [reply version_history.md]: I believe most of the comments in 2.7.1 still apply for 2.7.2

File: README.md (hunk 4 above)
Comment (ChristinaElder) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (ChristinaElder) [reply README.md]: updated

File: version_history.md (hunk 1 above)
Comment (ChristinaElder) [reply version_history.md]: Combined

File: version_history.md (hunk 2 above)
Comment (ChristinaElder) [reply version_history.md]: done

**Pull request merged**